packages/core/src/common/errors.ts

```diff
@@ -35,6 +35,7 @@ export const NUMERIC_INPUT_STEP_SIZE_NON_POSITIVE =
     ns + ` <NumericInput> requires stepSize to be strictly greater than zero.`;
 export const NUMERIC_INPUT_STEP_SIZE_NULL = ns + ` <NumericInput> requires stepSize to be defined.`;
 
+// TODO (clewis): Delete old Popover errors that aren't used anymore.
```

Comment:
can we get this resolved before merging?

Reply:
Yep

Reply:
@giladgray Turns out `Popover2` doesn't use _any_ of these. Seems like it should still handle these cases, so I'd be happy to leave these and file a follow-up issue. Thoughts?